internal/namespaces/baremetal/v1alpha1/custom_server_test.go

```diff
@@ -0,0 +1,50 @@
+package baremetal
+
+import (
+	"testing"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+func Test_StartServerErrors(t *testing.T) {
```

Comment:
Please name Test function after the Command they test (without the Errros suffix)